File: API/API/Model/Stat.cs
using System;
using System.Collections.Generic;

namespace API.Model;

public partial class Stat
{
    public uint StatId { get; set; }

    public ulong Nfollowers { get; set; }

    public ulong Nfollows { get; set; }

    public ulong Nsaves { get; set; }

    public ulong NyourSaves { get; set; }

    public virtual ICollection<User> Users { get; set; } = new List<User>();
}

File: shopmate_app/app/src/main/java/com/example/shopmate_app/model/api/Stat.kt
package com.example.shopmate_app.model.api

data class Stat (
    val statId : UInt?,
    val nfollowers : ULong,
    val nfollows : ULong,
    val nsaves : ULong,
    val nyourSaves : ULong
)

data class StatId(
    val statId: String
)


class Stats : ArrayList<Stat>()